_includes/upcoming-events.html

```diff
@@ -40,24 +41,24 @@ <h3 class="md:text-2xl text-base font-bold">{{ event.title }}</h3>
             >
 
             <!-- Read More / Read Less buttons -->
-            <a href="#" class="text-blue-400 read-more">Read More</a>
-            <a href="#" class="text-blue-400 read-less hidden">Show Less</a>
+            <button class="text-blue-400 read-more">Read More</button>
+            <button class="text-blue-400 read-less hidden">Show Less</button>
           </p>
 
           <p class="text-sm mb-1 text-gray-400 font-bold">
             {{ event.participants }}
           </p>
 
           <!-- Register Button -->
-          <a
-            href="{{ event.url }}"
-            class="inline-block bg-[{{site.bg-colors.orange-button}}] text-white font-semibold px-4 py-2 rounded-lg mt-2 hover:bg-[{{site.bg-colors.orange-button}}]/80"
-          >
-            Register Now
-          </a>
+          <button
+              class="inline-block bg-[{{site.bg-colors.orange-button}}] text-white font-semibold px-4 py-2 rounded-lg mt-2 hover:bg-[{{site.bg-colors.orange-button}}]/80 transition-colors duration-300"
+            >
+              Register Now
+            </button>
```

Comment:
It's better to keep the Register button as Anchor, since it should redirect the user to registration form(google form) directly through the href.

Reply:
sure, will look into it!